frontend/src/metabase/components/ErrorDetails/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface ErrorDetailsProps {
+  details: Record<string, any>;
+  centered: boolean;
+  className: string;
```

Comment:
Let's make `centered` and `className` optional.

And based on the usage of `details` in `ErrorDetails` around line 39, it looks like `details` is pretty loose. Maybe just make it `string | Record<string, any>` to account for the first predicate of the ternary (`typeof details === "string"`)